context/app/static/js/pages/Sample/Sample.jsx

```diff
@@ -35,15 +37,48 @@ function SampleDetail(props) {
     metadata,
     rui_location,
   } = assayMetadata;
+  const { elasticsearchEndpoint, nexusToken } = useContext(AppContext);
+
+  const { searchData: sampleSpecificDatasets } = useSearchData(
+    JSON.stringify({
+      query: {
+        bool: {
+          filter: [
+            {
+              term: {
+                ancestor_ids: uuid,
+              },
+            },
+            {
+              term: {
+                entity_type: 'dataset',
+              },
+            },
+          ],
+        },
+      },
+      _source: [
+        'uuid',
+        'display_doi',
+        'mapped_data_types',
+        'descendant_counts',
+        'origin_sample.mapped_organ',
+        'status',
+        'last_modified_timestamp',
+      ],
```

Comment:
This is just going to get the first 10 by default. I think we want to set a much larger page size.

Reply:
Added a size argument to the query, thanks!